hq/app/views/documentation/unknown.scala.html

```diff
@@ -0,0 +1,9 @@
+@(file: String)(implicit assets: AssetsFinder)
+
+<h2>Not found</h2>
+<p>No documentation found on "@file".</p>
+<p>Consider searching on
+    <a target="_blank" href="https://www.google.com/search?q=@file">Google</a>, or
```

Comment:
Same security issue here as in the other PR:

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

For the SGs and IAM links to AWS, several people asked it these could open in a new tab, so the knowledge may not be that common and it might be worthwhile keeping those as target blank links 😅

However, I think we should encourage people to discover ctrl click / cmd click. So I feel happy that these could default to opening in the same tab

Reply:
I'll take it out then.